uber/templates/emails/dealers/dealer_received.txt

```diff
@@ -1,11 +0,0 @@
-Dear {{ group.leader.full_name }},
```

Comment:
I did a search for this filename and it's not used anywhere in the codebase as far as I can tell.